edge/pkg/devicetwin/dtmodule/dtmodule_test.go

```diff
@@ -75,3 +80,135 @@ var _ = Describe("Dtmodule", func() {
 		})
 	})
 })
+
+func TestDTModule_InitWorker(t *testing.T) {
```

Comment:
TestDTModule_InitWorker and "Testing for InitWorker" context test look like duplicates

Reply:
I think this BDD test can be removed, ginkgo might be a bit heavy for unit testing

Reply:
Yeah it's duplicates, I realize it now.

Will remove BDD test.